synfig-studio/ui/synfig-catalog.xml

```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@Keyikedalube I thought this file is needed for Glade to work?

Reply:
@ice0 Oh! Did not notice that. Thanks for the second eye.

@DhairyaBahl That file is needed too. It's for synfig UI designers.

Reply:
@Keyikedalube Done :)

Reply:
Resolved.